### PR TITLE
Three new queries

### PR DESCRIPTION
**Pull request by NuriaQueralt**
Please @rajaram5 could you review these three new queries? bedankt

### CODE REVIEW

File: count-cysticFibrosis.rq [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#+ endpoint: http://ejprd.fair-dtls.surf-hosted.nl:8085/blazegraph/namespace/kb/sparql
```
Comment (rajaram5) [inline count-cysticFibrosis.rq]: @NuriaQueralt why do you name this query `count-cysticFibrosis`? Looks like this query return count for any resources right?

File: search-cysticFibrosis.rq [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#+ endpoint: http://ejprd.fair-dtls.surf-hosted.nl:8085/blazegraph/namespace/kb/sparql
```
Comment (rajaram5) [inline search-cysticFibrosis.rq:1]: Can we generalize this query to search for any given ordo URL?

**Review by rajaram5 — CHANGES_REQUESTED** [review]

File: count-cysticFibrosis.rq (hunk 1 above)
Comment (NuriaQueralt) [reply count-cysticFibrosis.rq]: @rajaram5 thanks for detecting it! Absolutely it is a mistake. This query counts the number of themes by biobank/patient registry. I will change the name.

File: search-cysticFibrosis.rq (hunk 2 above)
Comment (NuriaQueralt) [reply search-cysticFibrosis.rq:1]: Do you want me to create a new query or modify this one?

Comment (rajaram5) [reply search-cysticFibrosis.rq:1]: Can you modify it?

**Pull request merged**